frontend/src/pages/modelServing/screens/global/ModelServingGlobal.tsx

```diff
@@ -0,0 +1,15 @@
+import ApplicationsPage from 'pages/ApplicationsPage';
+import React from 'react';
+
+const ModelServingGlobal: React.FC = () => {
+  return (
+    <ApplicationsPage
+      title="Deployed models"
+      description="Manage and view the health and performance of your deployed models."
+      loaded
+      empty
+    ></ApplicationsPage>
```

Comment:
Why did you not render an empty content? `No Components Found` is not the best of wording.

Also, small nit, perhaps close the tag immediately or put `No content` in between the JSX tags.